### DISCUSSION TITLE
L'URL ne marche pas

### DISCUSSION THREAD

**Discussion by DJe73260** [Guilouz/Creality-Helper-Script-Wiki#564] [Issues]
Salut et bravo pour ton travail, très simple à mettre en oeuvre.
Toutefois, j'ai un soucis avec l'URL monIP:4408, qui me renvoie un "Ce site est inaccessible" alors qu'il marche sans le port 4408.

J'ai bien Fluidd et Moonraker d'installés dans les infos.
J'ai le meme pb avec Mainsail.

Dois je redémarrer ma K1 Max ??
Modèle : K1 Max
Firmware : 1.3.3.26 (Creality)

As tu une idée ???

![image](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/63497282/2eea4d98-f19d-4cd1-9a10-39ccecfaea57)

**Comment by Guilouz** [comment]
Essaye de relancer nginx.

**Reply by DJe73260** [reply]
Ok, comment fait on cela ?

Désolé de la question

**Reply by Guilouz** [reply]
Dans le menu tools du script

**Reply by DJe73260** [reply]
🙏💪👍

**Reply by DJe73260** [reply]
C'est fait. J'ai du tous les désionstaller et les réinstaller et ca marche maintenant..
Merci pour tout
